terraform/azure-ghaf-infra.tf

```diff
@@ -1,19 +1,45 @@
 # SPDX-FileCopyrightText: 2023 Technology Innovation Institute (TII)
 #
 # SPDX-License-Identifier: Apache-2.0
+terraform {
+  required_providers {
+    azurerm = {
+      source = "hashicorp/azurerm"
+    }
+    sops = {
+      source = "carlpett/sops"
+    }
+  }
+}
+provider "azurerm" {
+  features {}
+}
+# Sops secrets
+data "sops_file" "ghaf_infra" {
+  source_file = "secrets.yaml"
+}
+# Backend for storing tfstate (see ./azure-storage)
+terraform {
+  backend "azurerm" {
+    resource_group_name  = "ghaf-infra-storage"
+    storage_account_name = "ghafinfrastatestorage"
+    container_name       = "ghaf-infra-tfstate-container"
+    key                  = "ghaf-infra.tfstate"
+  }
+}
 # Resource group
 resource "azurerm_resource_group" "ghaf_infra_tf_dev" {
   name     = "ghaf-infra-tf-dev"
-  location = var.resource_group_location
+  location = "swedencentral"
```

Comment:
Is it necessary to add this hardcoding now? Before long we need this configurable (for the uae deployment).